pyproject.toml

```diff
@@ -38,7 +36,7 @@ exclude = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
-Django = ">=1.11,<3.1"
+Django = ">=2.2,<3.2"
```

Comment:
Would it be possible for us to open this further to include 3.2, on the basis that Django won’t be releasing breaking changes in a minor release?